app/src/main/java/com/randomappsinc/studentpicker/grouping/GroupingSettingsViewHolder.java

```diff
@@ -0,0 +1,69 @@
+package com.randomappsinc.studentpicker.grouping;
+
+import android.content.Context;
+import android.view.View;
+import android.widget.EditText;
+import android.widget.TextView;
+
+import com.randomappsinc.studentpicker.R;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
+class GroupingSettingsViewHolder {
+
+    @BindView(R.id.number_of_names_in_list) TextView numberOfNamesInList;
+    @BindView(R.id.num_of_names_per_group) EditText namesPerGroup;
+    @BindView(R.id.num_of_groups) EditText numGroups;
+
+    private Context context;
+    private GroupingSettings settings;
+
+    GroupingSettingsViewHolder(View view, Context context, GroupingSettings settings) {
+        ButterKnife.bind(this, view);
+        this.context = context;
+        this.settings = settings;
+        numberOfNamesInList.setText(
```

Comment:
Is the plan to replace 0 with an actual number later? I'm fine doing this in a later pull request, just wanted to make sure.

Reply:
yes, this is a reason for passing the context, you know in the dialog issue we spend a long time in it actually, I hate to pass the context because of the leak but I couldn't find a better way to set the text so, I kept it like this for now.